### PR TITLE
[FIX] Update postgresql version

### PR DESCRIPTION
**Pull request by lfreeke**

### CODE REVIEW

**Review by NL66278 — APPROVED** [review]
:+1: Remaining error is not related to travis.

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ env:
   matrix:
     - LINT_CHECK="1"
     - TESTS="1" ODOO_REPO="OCA/OCB"
-    - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+    - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="0"
```
Comment (astirpe) [inline .travis.yml]: According to https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L72 should be `MAKEPOT="0"` in the global stage:


```
env:
  global:
    - VERSION="11.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"

```

and should stay `MAKEPOT="1"` in test stage (https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L56)

**Review by astirpe — CHANGES_REQUESTED** [review]
Travis error is because the free Postcode API key is expired :(
The Postcode API module could be updated with a new key for the tests or alternatively the tests could be temporary disabled or rewritten.

File: .travis.yml (hunk 1 above)
Comment (lfreeke) [reply .travis.yml]: I solved this @astirpe

**Review by astirpe — APPROVED** [review]

**Comment by NL66278** [comment]
Add  EXCLUDE="l10n_nl_postcodeapi" to - TESTS="1" ODOO_REPO="OCA/OCB" and to the other line:
```
  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="l10n_nl_postcodeapi"
  - TESTS="1" ODOO_REPO="odoo/odoo"  MAKEPOT="1"  EXCLUDE="l10n_nl_postcodeapi"
```
Another reminder that tests should never rely on external services.

**Comment by lfreeke** [comment]
> Add EXCLUDE="l10n_nl_postcodeapi" to - TESTS="1" ODOO_REPO="OCA/OCB" and to the other line:
> 
> ```
>   - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="l10n_nl_postcodeapi"
>   - TESTS="1" ODOO_REPO="odoo/odoo"  MAKEPOT="1"  EXCLUDE="l10n_nl_postcodeapi"
> ```
> 
> Another reminder that tests should never rely on external services.

Done. All checks are green now

**Pull request merged**